diff --git a/CMakeLists.txt b/CMakeLists.txt
index 907b7b7d..0440ad34 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -135,7 +135,7 @@ if(WITH_RDP)
         PURPOSE "FreeRDP provides RDP support"
         TYPE REQUIRED
     )
-    find_package(WinPR)
+    find_package(WinPR 2.10)
     set_package_properties(WinPR PROPERTIES
         DESCRIPTION "Windows Portable Runtime"
         URL "https://www.freerdp.com/"
